### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by banzaman**
- Updated icons.
- Fixed errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -442,62 +442,74 @@ <h2 class="ski">Skills</h2>
...
+                  placeholder="Email address"
+                  class="user-email"
+                />
+              </li>
+              <li>
+                <textarea
+                  id="msg"
+                  placeholder="Enter text here"
+                  class="user-message"
+                ></textarea>
```
Comment (D4nielJ) [inline index.html:476]: - [ ] Let's validate your inputs. We can add `required` to them so the user can't submit the form without filling in the proper fields.

- [ ] Let's also add the attribute `maxlength` to the name and the message input, so the user can't spam in our fields. 30 characters for the name and 500 for the message.

File: index.html [hunk 2]
```diff
@@ -442,62 +442,74 @@ <h2 class="ski">Skills</h2>
...
+      <section>
+        <div class="last-part">
+          <article class="embuza">
+            I'm always<br />
+            interested in hearing<br />
+            about new projects,<br />
+            so if you'd like to<br />
+            chat please get in<br />
+            touch.
+          </article>
```
Comment (D4nielJ) [inline index.html:454]: [OPTIONAL] What do you think if we use the paragraph tag `p` this block of text?

File: index.html [hunk 3]
```diff
@@ -442,62 +442,74 @@ <h2 class="ski">Skills</h2>
...
+          <form
+            action="https://formspree.io/f/{form_id}"
```
Comment (D4nielJ) [inline index.html:456]: - [ ] Let's integrate correctly our form with [formspree service](https://formspree.io/html). For that, you need to make an account with them and then create a form. Once you have the endpoint link replace the link in `action` with it.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @banzaman, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥 

- You followed GitHub flow correctly
- You used flex correctly to align the items in your form.
- Your UI is looking great.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (banzaman) [reply index.html:456]: thank you

**Review by mikethreels — APPROVED** [review]
Hi Creed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**